turbinia/client.py

```diff
@@ -235,9 +237,61 @@ def get_task_data(
 
     return results[0]
 
+  def format_task_detail(self, task, show_files=False):
+    """Formats a single task in detail.
+
+    Args:
+      task (dict): The task to format data for
+      show_files (bool): Whether we want to print out log file paths
+
+    Returns:
+      list: Formatted task data
+    """
+    report = []
+    saved_paths = task.get('saved_paths') or []
+    status = task.get('status') or 'No task status'
+
+    report.append(fmt.heading2(task.get('name')))
+    line = '{0:s} {1:s}'.format(fmt.bold('Status:'), status)
+    report.append(fmt.bullet(line))
+    report.append(fmt.bullet('Task Id: {0:s}'.format(task.get('id'))))
+    report.append(
+        fmt.bullet('Executed on worker {0:s}'.format(task.get('worker_name'))))
+    if task.get('report_data'):
+      report.append('')
+      report.append(fmt.heading3('Task Reported Data'))
+      report.extend(task.get('report_data').splitlines())
+    if show_files:
+      report.append('')
+      report.append(fmt.heading3('Saved Task Files:'))
+      for path in saved_paths:
+        report.append(fmt.bullet(fmt.code(path)))
+      report.append('')
+    return report
+
+  def format_task(self, task, show_files=False):
+    """Formats a single task in short form.
+
+    Args:
+      task (dict): The task to format data for
+      show_files (bool): Whether we want to print out log file paths
+
+    Returns:
+      list: Formatted task data
+    """
+    report = []
+    saved_paths = task.get('saved_paths') or []
+    status = task.get('status') or 'No task status'
+    report.append(fmt.bullet('{0:s}: {1:s}'.format(task.get('name'), status)))
+    if show_files:
+      for path in saved_paths:
+        report.append(fmt.bullet(fmt.code(path), level=2))
+      report.append('')
+    return report
+
   def format_task_status(
       self, instance, project, region, days=0, task_id=None, request_id=None,
-      user=None, all_fields=False):
+      user=None, all_fields=False, full_report=False, priority_filter=20):
```

Comment:
Maybe have some constants for different priorities?
LOW = 80
MEDIUM = 50
HIGH = 10
CRITICAL = 0

Reply:
Yeah, that's a good idea.  I'll add that in somewhere.